modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java

```diff
@@ -159,6 +159,30 @@ public int getAvailableLocalesCount() throws Exception {
 		return availableLocales.size();
 	}
 
+	public String getBooleanTranslatedValues(String values, Locale locale) {
```

Comment:
@danvb112, let's review this pull request together today before or after our daily meeting. :wink:

Reply:
Ok @natocesarrego, Thanks! :wink: